context/app/static/js/components/Detail/provenance/ProvGraph/ProvGraph.jsx

```diff
@@ -3,48 +3,52 @@ import PropTypes from 'prop-types';
 
 import SectionItem from 'js/components/Detail/SectionItem';
 import { renderProvVis } from '../ProvVis';
-import { StyledTypography, StyledLink, FlexPaper } from './style';
+import { StyledTypography, FlexPaper } from './style';
 import '@hms-dbmi-bgm/react-workflow-viz/dist/react-workflow-viz.min.css';
 
 function ProvGraph(props) {
   const { provData } = props;
+  const isOld = 'ex' in provData.prefix;
+  const idKey = isOld ? 'hubmap:displayDOI' : 'hubmap:hubmap_id';
+  const timeKey = isOld ? 'prov:generatedAtTime' : 'hubmap:created_timestamp';
+  const typeKey = isOld ? 'prov:type' : 'hubmap:entity_type';
```

Comment:
Could make sense to store the details in an object in a config file or similar - otherwise looks good.

Reply:
Down the road, I just want to get rid of `*Key` ... They are really only needed during the transition. I'll file an issue for the cleanup (#1497) ... I don't think an outside config makes sense... it would be another thing to worry about switching between environments. (Though this attitude could be taken too far!)